lib/compute/agent-nodes.ts

```diff
@@ -6,9 +6,6 @@
  * compatible open source license.
  */
 
-import {
-  AmazonLinuxCpuType, AmazonLinuxGeneration, MachineImage,
-} from '@aws-cdk/aws-ec2';
```

Comment:
Please add back.

Reply:
Its not used anymore better to remove it.